### ISSUE TITLE
Comments should be placed above or behind code but not below

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#535]
We should issue a warning in case a comment is placed directly below some code, with an empty line following it.

Example:
```c#
var x = 42;
// it is 42 for some reason

DoSomething();
```

That comment describes the code part above, but it somehow looks like it may belong to the code below.

Instead, if the comment is placed behind then it is clear to which code the comment belongs to:
```c#
var x = 42; // it is 42 for some reason

DoSomething();
```

Same applies if the comment is placed above the code; it is clear where it belongs to:
```c#
// it is 42 for some reason
var x = 42;

DoSomething();
```

**Comment by RalfKoban** [comment]
The issue is already covered by following StyleCop Analyzer rules:
- SA1512 ("Single-line comments should not be followed by blank line")
- SA1515 ("Single-line comment should be preceded by blank line")
